language-extensions/python/build/windows/restore-packages.cmd

```diff
@@ -0,0 +1,46 @@
+SET ENL_ROOT=%~dp0..\..\..\..
+CALL %ENL_ROOT%\restore-packages.cmd
+nuget restore %ENL_ROOT%\language-extensions\python\packages.config -PackagesDirectory %ENL_ROOT%\packages
```

Comment:
We should remove the dependency on the internal Boost package in packages.config altogether - we need to find an alternative to External-Boost instead of that.

Reply:
How did we restore packages for these language extensions before here?